requirements.txt

```diff
@@ -19,4 +19,10 @@ albumentations>=1.3.0
 pytube>=12.1.3
 pyffmpeg==2.0
 GitPython>=3.1.18
+pytorch-metric-learning==1.6.3
```

Comment:
Is there a reason that the latest version won't work?

Reply:
Realized the necessary changes for pml 2.3.0 with the latest commit.